fv/test-workload/test-workload.go

```diff
@@ -302,6 +305,7 @@ func main() {
 			}
```

Comment:
I can't figure out how to add review comments outside of the diff here... but, just above here, can we update the comment: `// Listen on each port for either TCP or UDP.` ?